### PR TITLE
ReqresAPI

### PR DESCRIPTION
**Pull request by charam2k**

### CODE REVIEW

File: cypress/e2e/Reqres/reqres.cy.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      });
+    });
+
+    it('GET request for reqres', () => {
+      
+      cy.request("GET", "users/").then((response) => {
+        const filePath = 'cypress/downloads/response1.json';
+        expect(response.status).to.eq(200);
+        cy.writeFile(filePath, response.body);
+        //expect(response.body).to.have.property('name', 'cheri');
```
Comment (sreemayeN) [inline cypress/e2e/Reqres/reqres.cy.ts]: Remove the commented lines if you don't need them in future

File: cypress/downloads/response.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{
```
Comment (sreemayeN) [inline cypress/downloads/response.json]: Looks like this response file will be created once you've ran the code. You can remove them from the commit

File: cypress/downloads/response1.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (sreemayeN) [inline cypress/downloads/response1.json]: Looks like this response file will be created once you've ran the code. You can remove them from the commit

File: cypress.config.js [hunk 4]
```diff
@@ -5,7 +5,8 @@ module.exports = defineConfig({
     // setupNodeEvents(on, config) {
     //   // implement node event listeners here
     // },
-    baseUrl:"https://pokeapi.co/api/v2",
+   // baseUrl:"https://pokeapi.co/api/v2",
```
Comment (sreemayeN) [inline cypress.config.js:8]: Remove the commented duplicate base url

File: cypress/downloads/response1.json (hunk 3 above)
Comment (charam2k) [reply cypress/downloads/response1.json]: REsolved

**Pull request merged**